practices/mosaic-css.md

```diff
@@ -129,20 +127,20 @@ Common styles and patterns can then be reused across projects, and for Nature co
 
 #### Functional Class Names
-Use functional class names when the styling is based on their function or meaning. There is a strong connection between the class name, the styles it applies, and the reason the styles are being applied. Functional class names will always be used to describe OOCSS objects, as these are styles grouped together to carry out a particular function, and also for atomic positioning classes. Some examples include:
+Use functional class names when the styling is based on their function or meaning. There is a strong connection between the class name, the styles it applies, and the reason the styles are being applied. Functional class names will always be used to describe OOCSS objects, as these are styles grouped together to carry out a particular function, and also for positioning utility classes. Some examples include:
 
 ```css
 .pill-button {} /* OOCSS Object */
 
 .tab-group {}  /* OOCSS Object */
 
-.pin-left {} /* Atomic positioning class */
+.pin-left {} /*  positioning utility class */
 
-.position-absolute {} /* Atomic positioning class */
+.position-absolute {} /*  positioning utility class */
 ```
 
 #### Presentational Class Names
-Presentational class names describe the way an element looks. The name itself is describing the styles that are being applied. These classes are conducive to code reuse as they don't care what they are being used to style. This also comes with the benefit of scaling gracefully. As you're developing new components, you just need to add existing styles to your markup. Soon you will find that creating a new component requires no new css to be written, you are just applying existing styles to your HTML. Presentation class names are used for atomic styling classes. Some examples include:
+Presentational class names describe the way an element looks. The name itself is describing the styles that are being applied. These classes are conducive to code reuse as they don't care what they are being used to style. This also comes with the benefit of scaling gracefully. As you're developing new components, you just need to add existing styles to your markup. Soon you will find that creating a new component requires no new css to be written, you are just applying existing styles to your HTML. Presentation class names are used for utility styling classes. Some examples include:
```

Comment:
Awkward sentence: `These classes are conducive to code reuse as they don't care what they are being used to style.`

Upper case `CSS`.